content/src/tnips/tnip_2.md

```diff
@@ -0,0 +1,331 @@
+
+During epoch genesis, [CVVs](#committee-voting-validators-cvvs) that receive unexpected execution results must consider the most recent result. If a CVV receives an outdated execution result from a peer’s genesis certificate, then it responds with the correct execution results, including all signatures that formed a quorum. The CVV that included the fallback execution result must then verify all signatures of the more-recently signed execution result and reissue their genesis certificate to start the next epoch.
+
+This ensures that the closing executing result from the previous epoch will start the next epoch as long as at least one committee member receives the signed execution result in time.
+
+#### Preparing for the next epoch
+
+Validators that are in the next committee (C<sub>n</sub>) must update their peer lists to include all future committee members by making a request on the gossipsub peer network for peers matching the peer's BLS12-381 public keys.
+Nodes should drop peers in preference of their future committee members to ensure they have network information for all committee members by the start of the next epoch.
+
+### Eligible Nodes
+
+Telcoin Network is a proof-of-stake network (PoS) that requires proof of authority to stake.
+Authorities are GSMA full-members.
+
+#### Syncing
+
+New nodes synchronize by downloading all consensus output and executing the data up to the current epoch.
+The Telcoin Association's TAO manages snapshots of execution and consensus data to facilitate this process.
+Syncing must be permissionless and verifiable through independent execution.
+
+#### Staking Contract
+
+Validators must stake 1 million TEL to the designated staking contract at `0x07e17e17e17e17e17e17e17e17e17e17e17e17e1`. The locking period for stake is 10 epochs, after which a validator may withdraw their funds and exit the protocol entirely.
+
+#### NFT Requirement
+
+Validator wallets require an NFT issued by Telcoin Association for staking. The NFT issuance process involves a human, real-world review by the Telcoin Association and is exclusive to GSMA members.
+
+Validators must first obtain an NFT through Telcoin Association’s decentralized governance and have a fully synced node online. The validator NFT allows wallets to deposit TEL to the staking contract located at `0x07e17e17e17e17e17e17e17e17e17e17e17e17e1`.
+
+Once a node has completed the [staking process](#staking-contract), the validator's status is updated to "active" after one full epoch. Once the validator status is "active" on-chain, it is eligible to become a [CVV](#committee-voting-validators-cvvs). The newly eligible validator will be included in the next shuffle that determines C<sub>n+1</sub>. The node's effective stake is considered during the shuffle process.
+
```

Comment:
What does considering a node's effective stake mean?  Does this just mean a check to make sure they still have enough stake or are we going to allow validators to "buy" more committee time via stake?  Or maybe neither.

Reply:
I added this to the spec as something that needs to be further discussed through governance. In this context, it simply means the amount of stake that affects the node's chances of being selected during the shuffle process.